src/Neo/Network/P2P/RemoteNode.ProtocolHandler.cs

```diff
@@ -30,9 +30,9 @@ namespace Neo.Network.P2P
     partial class RemoteNode
     {
         private class Timer { }
-        private class PendingKnownHashesCollection : KeyedCollectionSlim<UInt256, (UInt256, DateTime)>
+        private class PendingKnownHashesCollection : KeyedCollectionSlim<UInt256, Tuple<UInt256, DateTime>>
```

Comment:
record?

Reply:
I'm trying to do this with minimum code changes.

Reply:
Then I think that this change is not required, isn't it?

Reply:
It is required. because `neo.io` is a `nullable` project. Also fix potential `NullException`.